### PR TITLE
Document restore force in nuget.exe and msbuild /t:restore

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/1729. 

fyi @tugruelmas

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[ddb058b](https://github.com/NuGet/docs.microsoft.com-nuget/commits/ddb058b56ab93b79d5639693d96ff2ca23305f06)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/cli-reference/cli-ref-restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/live-nkolev92-restoreForce/docs/reference/cli-reference/cli-ref-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-restore?branch=pr-en-us-1730) |
[docs/reference/msbuild-targets.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/live-nkolev92-restoreForce/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1730) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910082113213368-1730%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=js496lzYY3%2BqfnSDIdD1nOuaNp0w3YvlugjPX7Gd1YU%3D&st=2019-10-08T21%3A09%3A35Z&se=2019-11-08T21%3A14%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by nkolev92** [comment]
@NuGet/nuget-pm @NuGet/nuget-client 
👀

File: docs/reference/cli-reference/cli-ref-restore.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ where `<projectPath>` specifies the location of a solution or a `packages.config
 | RequireConsent | Verifies that restoring packages is enabled before downloading and installing the packages. For details, see [Package Restore](../../consume-packages/package-restore.md). |
 | SolutionDirectory | Specifies the solution folder. Not valid when restoring packages for a solution. Required when restoring with a `packages.config` file unless `PackagesDirectory` or `OutputDirectory` is used. |
 | Source | Specifies the list of package sources (as URLs) to use for the restore. If omitted, the command uses the sources provided in configuration files, see [Configuring NuGet behavior](../../consume-packages/configuring-nuget-behavior.md). Use a semicolon to separate list entries. |
+| Force | In PackageReference based projects, forces all dependencies to be resolved even if the last restore was succesfull. Specifying this flag is similar to deleting the project.assets.json file. This does not bypass the http-cache. |
```
Comment (karann-msft) [inline docs/reference/cli-reference/cli-ref-restore.md]:
```suggestion
| Force | In PackageReference based projects, forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is similar to deleting the project.assets.json file. This does not bypass the http-cache. |
```

File: docs/reference/msbuild-targets.md [hunk 2]
```diff
@@ -376,6 +376,7 @@ Additional restore settings may come from MSBuild properties in the project file
 | RestoreGraphProjectInput | Semicolon-delimited list of projects to restore, which should contain absolute paths. |
 | RestoreUseSkipNonexistentTargets  | When the projects are collected via MSBuild it determines whether they are collected using the `SkipNonexistentTargets` optimization. When not set, defaults to `true`. The consequence is a fail-fast behavior when a project's targets cannot be imported. |
 | MSBuildProjectExtensionsPath | Output folder, defaulting to `BaseIntermediateOutputPath` and the `obj` folder. |
+| RestoreForce | In PackageReference based projects, forces all dependencies to be resolved even if the last restore was succesfull. Specifying this flag is similar to deleting the project.assets.json file. This does not bypass the http-cache. |
```
Comment (karann-msft) [inline docs/reference/msbuild-targets.md]:
```suggestion
| RestoreForce | In PackageReference based projects, forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is similar to deleting the `project.assets.json` file. This does not bypass the http-cache. |
```

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[75f7349](https://github.com/NuGet/docs.microsoft.com-nuget/commits/75f734913b1cd9a8a44426018b03ad72e07528b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/cli-reference/cli-ref-restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/live-nkolev92-restoreForce/docs/reference/cli-reference/cli-ref-restore.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-restore?branch=pr-en-us-1730) |
[docs/reference/msbuild-targets.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/live-nkolev92-restoreForce/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/msbuild-targets?branch=pr-en-us-1730) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C15%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201910152137118880-1730%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yh0Ff1uCxO2SnFE99mfH5FBkMRf5GDDSX%2BX93OSV%2BT0%3D&st=2019-10-15T21%3A33%3A23Z&se=2019-11-15T21%3A38%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**